### DISCUSSION TITLE
Other Way to build OpenVINO from source other than Cmake ?

### DISCUSSION THREAD

**Discussion by Mr-MVP** [openvinotoolkit/openvino#22145] [Q&A] [answered]
I've been trying to build openvino from source using cmake but due to specification issues, my machine keeps restarting. Are there any other alternatives like makefile or ninja commands to build from source? 

Any input would be appreciated !

**Comment by ilya-lavrenov** [comment, marked as answer]
Cmake is a cross platform tool to generate build recipes for different build systems.
You can pass `-G` argument to cmake to generate a project for any supported build system. To generate project for Ninja, please use:
```bash
cmake -G Ninja -S <source dir> -B <build dir>
```
